py3status/parse_config.py

```diff
@@ -435,7 +435,7 @@ def make_function_value_private(self, value, value_type, function):
         if function == 'base64':
             try:
                 import base64
-                value = base64.b64decode(value)
+                value = base64.b64decode(value).decode()
```

Comment:
could we have an explicit `.decode('utf-8')` it just helps with reading the code